LanguageFeatures/Patterns/wildcards_A01_t01.dart

```diff
@@ -61,16 +61,16 @@ main() {
   Expect.equals(2, _two);
   Expect.equals(4, __);
 
-  var list2 = ["1", 2, "3", 4];
-  var [String _, two, _, num ___] = list2;
+  var list2 = [1, 2, 3.14, 4];
+  var [int _, two, _, num ___] = list2;
   Expect.equals(2, two);
   Expect.equals(4, ___);
 
   Expect.throws(() {
-    var [int _, _, _, num _] = list2;
+    var [int _, _, int _, num _] = list2;
   });
   Expect.throws(() {
-    var [_, String _, _, num _] = list2;
+    var [_, num _, _, num _] = list2;
```

Comment:
Not sure why this one would throw, what am I missing?

Reply:
Indeed. Removed this check. The one above is enough. Thank you